db/migrate/20240219143649_change_document_tags.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class ChangeDocumentTags < ActiveRecord::Migration[7.1]
+  def change
+    add_column :documents, :string_tags, :string, array: true, default: []
+
+    Document.find_each do |document|
+      document.tags.each { |tag| document.string_tags << tag }
+      document.save!(validate: false)
+    end
+
+    remove_column :documents, :tags, :jsonb, default: []
+    rename_column :documents, :string_tags, :tags
```

Comment:
Don't ever do this on a live app as it will die horribly 😂

Reply:
Is there a better way of doing it in that case? Or would we just have to suffer through it?

I assume neither adding nor removing columns is the issue, so is the problem renaming an existing one?